ravenframework/Models/PostProcessors/SparseSensing.py

```diff
@@ -93,12 +184,34 @@ def __init__(self):
     self.validDataType = ['PointSet','HistorySet','DataSet'] # FIXME: Should remove the unsupported ones
     self.sparseSensingGoal = None                            # The goal of the sensor selection. i.e., reconstruction or classification
     self.nSensors = None                                     # The number of the sensors required by the user.
+    self.nConstSensors = None                                # The number of sensors in the constrained region
+    self.constraintOption = None                             # The constraint name that the user wants to implement.
+    self.ConstrainedRegions = None                           # The shape of region we want to constrain
     self.nModes = None                                       # The number of modes/basis used to truncate the singular value decomposition
     self.basis = None                                        # The types of basis used in the projection. i.e., SVD, Identity, or Random Projection
     self.sensingFeatures = None                              # The variable representing the features of the data i.e., X, Y, SensorID, etc.
-    self.sensingTarget = None                                # The Response of interest to be reconstructed (or classify)
-    self.optimizer = None                                    # The Optimizer type using in the Sparse sensing selection (default: QR)
+    self.sensingStateVariable = None                         # The variable representing the state
+    self.sensingLabels = None                                # The Response of interest to be reconstructed (or classify)
+    self.optimizer = None                                    # The Optimizer type used in the Sparse sensing selection (default: QR)
+    self.classifier = None                                   # The classifier type used in the Sparse sensing selection (default: LinearDiscriminantAnalysis)
     self.sampleTag = 'RAVEN_sample_ID'                       # The sample tag
+    self.center_x = None                                     # The x co-ordinate of center of circle,ellipse
```

Comment:
Hm, it looks like this only supports one Constrained Region? I think PySensors supports multiple constrained regions?